lib/spack/spack/test/cmd/env.py

```diff
@@ -765,13 +765,13 @@ def noop(*args):
 @pytest.mark.usefixtures('config')
 def test_store_different_build_deps():
     r"""Ensure that an environment can store two instances of a build-only
-Dependency:
+    dependency::
 
-        x       y
-       /| (l)   | (b)
-  (b) | y       z2
-       \| (b)              # noqa: W605
-        z1
+              x       y
+             /| (l)   | (b)
+        (b) | y       z2
+             \| (b)
+              z1
```

Comment:
This seems to work pretty well:
![Screen Shot 2019-12-09 at 11 16 07 PM](https://user-images.githubusercontent.com/12021217/70497805-05c1c880-1ada-11ea-9321-e7a5dae718d5.png)